tests/nightwatch/responsive-test-suite-spec.js

```diff
@@ -16,6 +18,6 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
       // eslint-disable-next-line no-console
         throw new Error(`The test breakpoint width is: ${testWidth} and the browser width is ${result.value.width}`);
       }
-    });
+    }).perform(snapshot);
```

Comment:
We should add an addionaly parameter to accept a screenshot description. When one isn't provided, screenshot name is 'Default'.

Reply:
yeah, need to update this to take options.. it'll likely look like

`.perform(snapshot(name, {}))`